### DISCUSSION TITLE
群组中，父节点哪些行为会同步到子节点

### DISCUSSION THREAD

**Discussion by drl990114** [antvis/X6#1693] [Q&A]
当组成群组后，除了位置移动是一起的，父节点的缩放和旋转不会对子节点产生影响，这个是正常的吗？
[https://codesandbox.io/s/red-grass-3fqdh?file=/src/app.tsx](https://codesandbox.io/s/red-grass-3fqdh?file=/src/app.tsx)

**Comment by NewByVector** [comment]
在 drawio 中旋转父节点是会同时旋转子节点的。
